.travis.yml

```diff
@@ -19,6 +19,6 @@ before_install:
     - git submodule update --init --remote --recursive
 
 install:
-  - git clone --depth=1 https://github.com/ros-industrial/industrial_ci.git .industrial_ci
+  - git clone --depth=1 --branch legacy-writable https://github.com/oscar-lima/industrial_ci.git .industrial_ci
```

Comment:
I am not sure this will ever be fixed, check [issue](https://github.com/ros-industrial/industrial_ci/pull/123), ipa mdl qutoe: "I really don't like making the src writable again, as the build should never mess with the src folder" . Our options are: 1) maintain a fork of industrial CI under KCL account (currently is on my account which I don't like so much) 2) Modify ROSPlan code to not create files under src. We could either create them on /var/tmp or ~/.ros

Reply:
Which files are created? I can think of only the plan and problem files at the moment. We could change the launch parameter for those paths to point towards devel or ~/.ros.

Reply:
yes, problem and plan. It would be cleaner to store them somewhere else, agree with that. Then we wouldn't need the industrial CI fork. Will work on it and improve then this PR

Reply:
Agree that they should not be in the src file. That may need a different PR though, and then we can fix the ci back to the official one?
I believe that even for the user it's weird to find random problem files around the src, more when they're leftover from some other time. /tmp sounds a good idea for me, autocleaned after each boot. Generated problem files should not be kept anyway right?

Reply:
yes, agree. If this is a change we want to make in another PR, then #200 is ready to be merged, if not then I can make the required changes in #200 over the weekend.

Reply:
I think is better to leave it in another PR as it's not related directly to testing. Let's just not forget it.